### PR TITLE
documentation for util.coffee

### PR DESCRIPTION
**Pull request by SteveMcArthur**

### CODE REVIEW

File: src/lib/util.coffee [hunk 1]
```diff
@@ -13,45 +13,102 @@ extractOptsAndCallback = require('extract-opts')
 
 # =====================================
 # Export
+###*
+# The DocPad Util Class.
+# Collection of DocPad utility methods
+# @class docpadUtil
+# @constructor
```
Comment (balupton) [inline src/lib/util.coffee]: Is this the correct properties for a standard javascript object? As it is not a class or a constructor.

**Comment by SteveMcArthur** [comment]
Yeah - I had trouble here. If you scroll down a bit you will see that I have labeled it as static. I think what you have created here is a static class - ie a once only class. Of course JS doesn't really have any concept of class - but to all intents and purposes a once only class is being created here. And its a constructor because you are creating an object. I think for the average user this will make sense in the documentation.

**Comment by SteveMcArthur** [comment]
@balupton my suggestion here is to just push up the merge. If enough people disagree with my view then we can easily change the documentation. I will happily do the change.

**Comment by balupton** [comment]
Possibly, it can be revisited if someone knows a better way of mapping it in YUIDoc or when we migrate to ES6+. Merging.

**Pull request merged**